modules/ROOT/pages/visual-tour/index.adoc

```diff
@@ -24,17 +24,28 @@ image::breadcrumbs.png[Breadcrumb navigation]
 
 == Organizations
 
-Organizations are a layer in the Aura account hierarchy that sits above Projects.
+Organizations are a layer in the Aura account hierarchy that sits above projects.
 This allows you to organize and manage multiple projects under a single organization.
 
-Management at this level allows control over all Projects within an Organization and instances within Projects and lets `Organization owners` roll out Project-level settings for all Projects simultaneously.
+Management at this level helps `Organization owners` and `Organization admins` roll out settings to all projects and instances within an organization.
```

Comment:
```suggestion
Management at this level allows `Organization owners` and `Organization admins` roll out settings to all projects and instances within an organization.
```

Reply:
Here, I'd suggest "Management at this level allows `Organization owners` and `Organization admins` roll out settings to projects and instances within an organization."

Just to de-emphasize how many projexts you can have.